CHANGELOG.adoc

```diff
@@ -54,6 +54,7 @@
 - https://github.com/eclipse-sirius/sirius-web/issues/3550[#3550] [core] Add ExtensionRegistry merge strategy
 - https://github.com/eclipse-sirius/sirius-web/issues/3563[#3563] [sirius-web] Improve NavigationBar extensibility to allow the contribution of components on the left and right of the navigation bar.
 - https://github.com/eclipse-sirius/sirius-web/issues/3344[#3344] [core] Add support for reloading representations from the database
+- https://github.com/eclipse-sirius/sirius-web/issues/3557[#3553] [core] Add RepresentationFactory extension point
```

Comment:
I'll add an entry to indicate that a breaking change is being done in this PR with the removal of RepresentationContext